.vscode/settings.json

```diff
@@ -24,5 +24,7 @@
     ],
     "git.branchProtection": [
         "main"
-    ]
+    ],
+    "javascript.preferences.importModuleSpecifier": "relative",
+    "typescript.preferences.importModuleSpecifier": "relative"
```

Comment:
I'm not sure we want to enable this for all users? I can't speak for DCR, but AR generally doesn't use relative imports.

Reply:
Oh, I thought this was true for the whole codebase, thanks for flagging, @JamieB-gu!